### ISSUE TITLE
No separate tee skin prefix for dummy

### ISSUE THREAD

**Issue by Kruimeldief** [ddnet/ddnet#5557]
When choosing a tee skin prefix (`none`, `kitty`, `santa`), it's applied to the main tee as well as the dummy tee regardless of your _dummy setting_ checkbox selection.

And something else: an insignificant cosmetic bug is that the tee skin changes aren't applied when closing the menu in some cases. When this happens, you need to re-open the tee skin settings, change something (e.g. toggle checkbox twice) and close the menu. This happens most often when changing the tee skin settings of the opposite selected tee (e.g. dummy settings when focused on main tee).

**Comment by heinrich5991** [comment]
The skin prefix thing is intentional, it's supposed to override all skins, not only your own.

**Comment by Kruimeldief** [comment]
> The skin prefix thing is intentional, it's supposed to override all skins, not only your own.

Oh, that's too bad. I thought I used the skin prefix to differentiate my main and dummy tee by their ears (or no ears) with the same vanilla skin. But after browsing the non-vanilla skins, I noticed there's a similar skin which I probably used before*.

* I had to reconfigure the client settings due to accidentally resetting them by executing a cleanly downloaded DDNet.
